test/forms/nested-repeats.xml

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0"?>
```

Comment:
This is okay, but I would normally use a much much smaller form, so keep these tests easier to understand. E.g. no itext, only 1 or 2 select options (or just text inputs).

Reply:
tried to simplify the form